src/server/index.tsx

```diff
@@ -361,3 +293,116 @@ async function fetchIconPng(iconUrl: string) {
     .then(res => res.blob())
     .then(blob => blob.arrayBuffer());
 }
+
+function getErrorPageData(error: string, site?: GetSiteResponse) {
+  const errorPageData: ErrorPageData = {};
+
+  // Exact error should only be seen in a development environment. Users
+  // in production will get a more generic message.
+  if (process.env.NODE_ENV === "development") {
+    errorPageData.error = error;
+  }
```

Comment:
Im not sure this makes sense. The reason not to show error messages is usually for security, but anyway the error messages are available if you use the API directly. 

Another reason might be to keep things simple for non-techie users. But even if they dont understand the error, there can be a text telling them to copy-paste the error message to admins/devs. That can be very helpful for debugging.